src/azure-cli/azure/cli/command_modules/privatedns/tests/latest/test_privatedns_commands.py

```diff
@@ -778,9 +778,8 @@ def test_PatchRecordSet_RemoveMetadata_ExpectMetadataRemoved(self, resource_grou
             self.check('name', '{recordset}'),
             self.check('metadata.{}'.format(tagKey), tagVal)
         ])
-        self.cmd('az network private-dns record-set a update -g {rg} -n {recordset} -z {zone} --metadata ""', checks=[
-            self.check('name', '{recordset}'),
-            self.check('metadata', '{{}}')
+        self.cmd('az network private-dns record-set a update -g {rg} -n {recordset} -z {zone} --metadata null', checks=[
```

Comment:
So the `""` is not supported and customer should use `null` instead right? If so we should add it in the change log.